test/integration/testLargeResultSet.js

```diff
@@ -113,4 +113,126 @@ describe('Large result Set Tests', function ()
       }
     });
   });
+
+  it('testVariantLarge', function (done)
+  {
+    const createTableWithVariant = 'create or replace table testVariantTable(colA variant)';
+    const insertVariant = 'insert into testVariantTable select value from table(flatten(parse_json(?)))';
```

Comment:
We are sending one gigantic array in json and let Snowflake to parse it and store as one json per row. Why don't we send binds as array of arrays? is it important for the test?

Reply:
This is how we were advised by the previous engineer when inserting multiple rows of variant data

I tried binding an array of arrays but got `OperationFailedError: SQL compilation error: error line 1 at position 53 Bind variable ? not set.`
```
sqlText: "insert into testVariantTable(colA) select parse_json(?)",
binds: [ [JSON.stringify({ 'root': 'value' })], [JSON.stringify({ 'root': 'value' })] ],
```

I then tried using insert into values but got a `Error [OperationFailedError]: SQL compilation error: Unsupported data type 'VARIANT'.` instead
```
sqlText: "insert into testVariantTable(colA) values(?)",
binds: [ [{ 'root': 'value' }], [{ 'root': 'value' }] ],
```

However, that error did not show up when array binding non-variant data
```
sqlText: 'insert into testTable values(?)',
binds: [[1], [2], [3]],
```

And running the first query without array bind was successful
```
sqlText: 'insert into testVariantTable(colA) select parse_json(?)',
binds: [JSON.stringify({ 'key': 'value' })],
```

Is there a different combination of sql and binds that I should be using instead?